curated_transformers/pipe.py

```diff
@@ -206,16 +235,19 @@ def predict(self, docs: Iterable[Doc]) -> TransformerModelOutput:
         return self.model.predict(docs)
 
     def set_annotations(
-        self, docs: Sequence[Doc], trf_output: TransformerModelOutput
+        self, docs: Sequence[Doc], predictions: TransformerModelOutput
```

Comment:
I think I like the original name better. _predictions_ sound like things like class labels.

Reply:
Reverted.